src/OpenTelemetry.AutoInstrumentation/Instrumentations/Log4Net/Integrations/AppenderCollectionIntegration.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using OpenTelemetry.AutoInstrumentation.CallTarget;
+
+namespace OpenTelemetry.AutoInstrumentation.Instrumentations.Log4Net.Integrations;
+
+/// <summary>
+/// Log4Net AppenderCollection integration.
+/// </summary>
+[InstrumentMethod(
+assemblyName: "log4net",
+typeName: "log4net.Appender.AppenderCollection",
+methodName: "ToArray",
+returnTypeName: "log4net.Appender.IAppender[]",
+parameterTypeNames: new string[0],
+minimumVersion: "2.0.0",
+maximumVersion: "3.*.*",
```

Comment:
Just double check, there is no changes between 2.0.0 and 3.0.0 in instrumented method?

Reply:
That's correct.